ui/src/branch-actions/actions/inspect-conflicts.tsx

```diff
@@ -35,12 +38,21 @@ function InspectConflicts({ branches }: ActionComponentProps) {
 	const conflictDetails = useSuspenseQuery(
 		queries.getConflictDetails(branches.map((b) => b.name)),
 	).data;
+	const branchNames = branches.map((b) => b.name);
 
 	return (
 		<>
-			TODO
-			{t('see-conflicts')}
-			{JSON.stringify(conflictDetails)}
+			<Prose>
+				{t('conflict-count', { count: conflictDetails.conflicts.length })}
```

Comment:
Using the count here is cool and new to me, is there react i18next documentation on this I could look up? I haven't found it yet.

Reply:
You can find that here: https://www.i18next.com/translation-function/plurals

Reply:
Found it: https://www.i18next.com/translation-function/plurals